src/shared/test/integration/inter-frame-communication-test.js

```diff
@@ -0,0 +1,241 @@
+import { delay } from '../../../test-util/wait';
+import { Bridge as BridgeOri } from '../../bridge';
```

Comment:
I presume `Ori` is short for `Original` here? We don't have a lot of precedents for this, but appending an underscore (`Bridge_`) or using an `Orig` prefix (`OrigBridge`) are more common approaches in our code.